packages/mobile/src/account/Settings.tsx

```diff
@@ -227,101 +201,123 @@ export class Account extends React.Component<Props, State> {
     }
   }
 
+  disableFornoMode = () => {
+    this.props.toggleFornoMode(false)
+    this.hideFornoSwitchOffWarning()
+  }
+
+  handleFornoToggle = (fornoMode: boolean) => {
+    if (!fornoMode && this.props.gethStartedThisSession) {
+      // Starting geth a second time this app session which will
+      // require an app restart, so show restart modal
+      this.showFornoSwitchOffWarning()
+    } else {
+      this.props.toggleFornoMode(fornoMode)
+    }
+  }
+
+  showFornoSwitchOffWarning = () => {
+    this.setState({ fornoSwitchOffWarning: true })
+  }
+
+  hideFornoSwitchOffWarning = () => {
+    this.setState({ fornoSwitchOffWarning: false })
+  }
+  onPressPromptModal = () => {
+    this.props.toggleFornoMode(true)
+    navigateBack()
+  }
+
+  hidePromptModal = () => {
+    this.props.toggleFornoMode(false)
+    navigateBack()
+  }
+
+  goToTerms() {
```

Comment:
Can we keep to a common convention for these handler names? Ideally they should begin with `on`

Reply:
Maybe not your code though, I understand